spec/views/users/show.html.erb_spec.rb

```diff
@@ -1,5 +1,73 @@
 require 'rails_helper'
 
-RSpec.describe 'users/show.html.erb', type: :view do
-  pending "add some examples to (or delete) #{__FILE__}"
+RSpec.describe 'Renders users Show Page', type: :feature do
+  before(:each) do
+    # Create necessary objects for the tests, such as a user and their posts
+    @user = User.create(id: 1, name: 'Marc', photo: 'profile.jpg', bio: 'Software Engineer', posts_counter: 10)
+    @first_post = Post.create(author: @user, title: 'Hi people', text: 'Lorem ipsum dolor sit amet.')
+    Post.create(author: @user, title: 'Good', text: 'Consectetur adipiscing elit.')
+    Post.create(author: @user, title: 'Dev', text: 'Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.')
+    @post = Post.create(author: @user, title: 'Hello', text: 'Lol')
+  end
+
+  scenario 'Display user profile picture' do
+    visit user_path(@user.id)
+    expect(page).to have_css("img[src*='profile.jpg']")
+  end
+
+  scenario 'Display user username' do
+    visit user_path(@user.id)
+    expect(page).to have_content('Marc')
+  end
+
+  scenario 'Display number of user posts' do
+    visit user_path(@user.id)
+    expect(page).to have_content('Number of posts: 14')
+  end
+
+  scenario 'Display user bio' do
+    visit user_path(@user.id)
+    expect(page).to have_content('Software Engineer')
+  end
+
+  # ################################################################*************************************
+
+  scenario 'check if you can see who wrote the post' do
+    visit user_post_path(@user, @first_post) ##### please approve it 😌
+    expect(page).to have_content(@first_post.author.name)
+  end
+
+  # ################################################################***************************************
```

Comment:
Nice job adding this test :+1: However, according to the [requiremets](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Post%20show%20page,each%20commentor%20left.), it should be added in the spec for the `post#show` page (`posts/show.html.erb_spec.rb`). Kindly make the appropriate changes to make this happen.

Reply:
Done! :c